src/fixup.js

```diff
@@ -139,4 +140,78 @@
     }
   }
 
+  /* Deprecation warning */
+  var request = new XMLHttpRequest();
+  request.open('GET', '//www.w3.org/TR/tr-outdated-spec.json', true);
```

Comment:
Nit: you don't need "true" (as it's the default.)

Reply:
😱😱😱, sorry, strong objection here. That JSON file is 400KB! Making everyone download 400kb every time they view a spec is probably not ok.

Reply:
My suggestion is that we have a lookup service instead. 

```
GET //www.w3.org/TR/tr-outdated-spec
Referrer: https://ww.w3.org/TR/SomeSpec
```

And then return 200 OK if it's outdated, and no entity body. Otherwise, return 404.

Reply:
Actually, you could return the location of the latest. You get the logic below for free.

Reply:
See suggestions below about what the actual body could be... but effectively localized text that would look like:

```JSON
{
    "newSpec": "new spec location",
    "warning":  ".... bla bla ...."
}
```